modules/acscs-secured-cluster-services-overview.adoc

```diff
@@ -5,6 +5,7 @@
 [id="acscs-secured-cluster-services_{context}"]
 = Secured cluster services
 //when updating this page, if applicable, update the corresponding page for ACS - acs-secured-cluster-services-overview.adoc
+//make sure the product name in this file is {product-title-managed-short}.
 
 You install the secured cluster services on each cluster that you want to secure by using the {product-title-managed-short}.
```

Comment:
@gaurav-nelson, FYI, lines 8 and 10 seem to contradict each other.

Reply:
I agree with Max. Is a conditional needed here to differentiate the product attributes?

Reply:
I wonder if it would make sense to move such comment lines up and append them to the first three comment lines at the top of this file. Also, imperatives tend to be longer, so this info can be shortened by using statements. For example:
```
// Module included in the following assemblies:
//
// * cloud_service/acscs-architecture.adoc
//
//This topic is about {product-title-managed-short} and tied to acs-secured-cluster-services-overview.adoc